model/minio/minio_cluster.rb

```diff
@@ -13,6 +13,11 @@ class MinioCluster < Sequel::Model
   many_to_one :private_subnet, key: :private_subnet_id
 
   include ResourceMethods
+
+  def self.redacted_columns
+    super + [:root_cert_1, :root_cert_2]
```

Comment:
It automatically redacts encrypted keys.
https://github.com/ubicloud/ubicloud/blob/cc585441c9a7a8557a7f6fdd9c8c49c8818258e0/model.rb#L120

Reply:
`root_cert_1` and `root_cert_2` are not encrypted.